scenarios/Assistants/api-in-a-box/wind_farm/assistant-wind_farm.ipynb

```diff
@@ -4,9 +4,57 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Wind Turbine Farm Management Assistant\n",
+    "# Wind Turbine Farm Management Assistant"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Objective\n",
     "\n",
-    "### Load the required libraries"
+    "{{TODO: Add a one or two sentences describing the learning outcomes for this notebook}}\n",
+    "\n",
+    "This tutorial uses the following Azure AI services:\n",
+    "\n",
+    "- {{TODO: Add short descriptions and links to documentation for Azure AI services used}}\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Time\n",
+    "\n",
+    "You should expect to spend 5 minutes running this sample."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## About this example\n",
+    "\n",
+    "{{TODO: Include a paragraph or two explaining what this example demonstrates, who should be interested in it, and what you need to know before you get started.}}\n",
+    "\n"
```

Comment:
Needs a description

Reply:
# Wind Turbine Farm Management Assistant

## Objective
This notebook demonstrates the following:

- Utilizing Assistant tools such as the Code Interpreter and Function calling, this bot is capable of retrieving a CSV file that illustrates turbine wind speed, voltage, and the last maintenance date. It assists you in reviewing the file contents and aids in determining whether a specific turbine is in need of maintenance.

This tutorial uses the following Azure AI services:
- Access to Azure OpenAI Service - you can apply for access [here](https://aka.ms/oai/access)
- Azure OpenAI service - you can create it from instructions [here](https://learn.microsoft.com/en-us/azure/ai-services/openai/how-to/create-resource)
- Azure OpenAI Studio - go to [https://oai.azure.com/](https://oai.azure.com/) to work with the Assistants API Playground
- A connection to the Azure OpenAI Service with a [Key and Endpoint](https://learn.microsoft.com/en-us/azure/ai-services/openai/chatgpt-quickstart)

Reference:
- Learn more about how to use Assistants with our [How-to guide on Assistants](https://learn.microsoft.com/en-us/azure/ai-services/openai/how-to/assistant).
- [Assistants OpenAI Overview](https://platform.openai.com/docs/assistants/overview)

## Time
You should expect to spend 5-10 minutes running this sample. 

## About this example
This sample demonstrates the creation of an Azure OpenAI Assistant named "Turbine Management Assistant" utilizing the Azure OpenAI API. The assistant functions as a management tool for wind turbine farms, providing data, charts, and actionable insights regarding the status and performance of each turbine within the farm. The notebook/script orchestrates a dialogue with the assistant, navigating through different inquiries and scenarios to demonstrate its capabilities.

### Data
This sample uses files from the folder [`data/energy/`](../data/energy/) in this repo. You can clone this repo or copy this folder to make sure you have access to these files when running the sample.

Reply:
Resolved by https://github.com/Azure-Samples/azureai-samples/pull/59/commits/c98eaac6cf98fb54da5eb00215481f3fa0e2d6cb